subprojects/docs/docs.gradle

```diff
@@ -309,25 +318,6 @@ samples {
                 from(templates.javaUtilitiesLibraryAsSubproject)
             }
         }
-        javaApplication {
-            sampleDirectory = samplesRoot.dir("java/application")
-            description = "Build a Java application."
-            category = "Java"
-            common {
-                from(templates.javaApplicationInApp)
-                from(templates.javaListLibraryInApp)
-                from(templates.javaUtilitiesLibraryInApp)
-            }
-        }
-        javaLibrary {
```

Comment:
⭕ So we get rid of the `javaApplication` sample in favour of the samples migrated from the guides?

Reply:
Yes.

Reply:
👍